content/training/R_Basics/Advanced_Functions.Rmd

```diff
@@ -19,17 +19,21 @@ linelist_cleaned <- gen_data("AJS")
 ```
 
 
-### Factors and the forcats() package  
+### Factors and the *forcats* package  
 
 #### Factors
 
 Factors are a special variable class in which data are stored internally as integers, but each integer *level* has a corresponding character value *label* that is used when the data are displayed. This can be partularly helpful when running statistical models which expect numeric (e.g. 0/1) inputs.  
 
-*Factors are similar to labeled values of categorical variables in Stata.*  
+{{% notice tip %}}
```

Comment:
This tip looks really big (tall) in the demo site... is that normal?

Reply:
no that is not normal (le sigh)

Reply:
I know why. It's because I put spaces between the paragraph and the notice tag :man_facepalming:

Reply:
okay, should be fixed as of e2ce7fa